Bottom: 6d16fb9aaf3f2d6163ad1ef73fc3ba27d741bb20
Top:    d7f0989db31ffd408e32b8aabb0d47c32a146120
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-22 12:01:07 +0000

Remove "=" shortcut to the djopt() thingy which keeps eating my design

ARR!! how much work have I lost to this key binding!










---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res
index 59a9a29..7843ca4 100644
--- a/src/gpcb-menu.res
+++ b/src/gpcb-menu.res
@@ -332,13 +332,11 @@ MainMenu =
    {"Rip up all auto-routed tracks" RipUp(All)}
    -
    {"Optimize routed tracks"
-    {"Auto-Optimize" djopt(auto)  a={"Shift-=" "Shift<Key>="}}
     {"Debumpify" djopt(debumpify) }
     {"Unjaggy" djopt(unjaggy) }
     {"Vianudge" djopt(vianudge) }
     {"Viatrim" djopt(viatrim) }
     {"Ortho pull" djopt(orthopull) }
-    {"Simple optimization" djopt(simple)  a={"=" "<Key>="}}
     {"Miter" djopt(miter) }
     {"Puller" a={"Y" "<Key>y"} Puller() }
     {"Global Puller"
